train.py

```diff
@@ -24,6 +24,9 @@
     [int, 'n_layers_before_tf', 0, 'when using mix framework, number of layer defined using upstride', lambda x: x >= 0],
     [str, 'load_searched_arch', '', 'model definition file containing the searched architecture'],
     [str, "model_name", '', 'Specify the name of the model', lambda x: x in model_name_to_class],
+    [bool, "use_wandb", False, 'enable if we want to utilize weights and biases'],
+    [str, 'project', 'project0', 'Unique project name within which the training runs are executed in wandb',],
+    [str, 'run_name', '', 'Unique run name within which the training runs are executed in wandb',],
```

Comment:
these 3 new parameters should be put in a namespace to keep a clean organization

Reply:
agreed. Updated the same

Reply:
I think you defined the namespace but you forgot to update the code when you call the argument

Reply:
Its fixed now and tested. Sorry